gammapy/image/tests/test_mask.py

```diff
@@ -16,7 +16,32 @@ def test_random_creation():
     assert excluded[0].size != 0
 
 
-@pytest.mark.xfail
+@requires_dependency('scipy')
 def test_distance_image():
-    pass
-    # TODO
+    mask = ExclusionMask.empty(nxpix=3, nypix=2)
+    skymap = mask.distance_image
+    distance = skymap.data
+    assert distance.shape == (2, 3)
+    assert_allclose(distance[0, 0],  -1.)
+    assert_allclose(distance[-1, 0], -2.)
+    assert_allclose(distance[0, -1], -2.23606798)
+    assert_allclose(distance[-1, -1], -2.82842712)
```

Comment:
The results for input of `mask` of all zeros is still weird.
It measures the distance to some edge or corner, which never makes sense for our applications.

Can you also check for that and in that case return `-1e10`?
This has the meaning "we're assuming we are deep inside an exclusion region".

OK?